### PR TITLE
Real progress

### PR DESCRIPTION
**Pull request by ptmkl**
Implemented being able to see the count of annotated images for each imageset on the home page and team pages

### CODE REVIEW

File: imagetagger/imagetagger/images/views.py [hunk 1]
```diff
@@ -92,6 +92,16 @@ def index(request):
...
+    
+    # add annotated image count to each imageset
+    for imageset in imagesets:
+        images = Image.objects.filter(image_set=imageset)
+        annotations = Annotation.objects.filter(image__in=images,
+                                            annotation_type__active=True).select_related()
+        annotatedimages = set()
+        for annotation in annotations:
+            annotatedimages.add(annotation.image)
+        imageset.annotated_image_count = len(annotatedimages)
```
Comment (WasabiFan) [inline imagetagger/imagetagger/images/views.py]: Nice! FWIW I think you can somewhat shorten this with a Python generator statement:

```python
imageset.annotated_image_count = len(set((annotation.image for annotation in annotations)))
```

It's the same thing just written on one line using a generator statement thing.

File: imagetagger/imagetagger/users/views.py [hunk 2]
```diff
@@ -250,6 +250,16 @@ def view_team(request, team_id):
     export_format_forms = (ExportFormatEditForm(instance=format_instance) for format_instance in export_formats)
     test_imagesets = imagesets.filter(set_tags__name='test').order_by('-public', 'name')
 
+    # add annotated image count to each imageset
+    for imageset in imagesets:
+        images = Image.objects.filter(image_set=imageset)
+        annotations = Annotation.objects.filter(image__in=images,
```
Comment (WasabiFan) [inline imagetagger/imagetagger/users/views.py:271]: I'd normally object to the two copies of this code but it seems like that's the norm in this codebase 🤷‍♂

**Review by WasabiFan — APPROVED** [review]
Good stuff! Minor comments, you can choose what's worth doing. Can you edit the PR title to describe the feature a bit? (e.g. "Implement ...")

File: imagetagger/imagetagger/users/views.py (hunk 2 above)
Comment (ptmkl) [reply imagetagger/imagetagger/users/views.py:271]: haha yeah i think thats just how django is

File: imagetagger/imagetagger/images/views.py (hunk 1 above)
Comment (ptmkl) [reply imagetagger/imagetagger/images/views.py]: fixed!

**Pull request merged**